NAMESPACE

```diff
@@ -75,8 +77,14 @@ importFrom(graphics,plot.new)
 importFrom(graphics,title)
 importFrom(limma,squeezeVar)
 importFrom(lme4,lmer)
+importFrom(magrittr,"%>%")
```

Comment:
I'd rather avoid this

Reply:
Commit [remove %>%](https://github.com/Vitek-Lab/MSstats/pull/107/commits/26c20f687fbd0231bf9687965ac3ff4c5ffd3518)